pkg/store/string.go

```diff
@@ -155,6 +220,7 @@ func (s *Store) Set(db uint32, args [][]byte) error {
 
 	o := newStringRow(db, key)
 	o.Value = value
+	o.ExpireAt = expireat
```

Comment:
if no EX or PX in set cmd,  expireat should be original o.ExpireAt value not int64(0)?